integration-tests/otlp/src/main/java/io/opentelemetry/integrationtest/OtlpExporterIntegrationTest.java

```diff
@@ -435,6 +442,7 @@ private static void testMetricExport(MetricExporter metricExporter) {
                     .setKey("key")
                     .setValue(AnyValue.newBuilder().setStringValue("value").build())
                     .build()));
+    assertThat(dataPoint.getExemplarsCount()).isGreaterThanOrEqualTo(0);
```

Comment:
Are we expecting to see at least 1 exemplar?

Reply:
Ooo yes. Meant to say "isGreaterThan(0)`, but think its ok to be more precise. Fixed assertion in new commit.